start-minecraftFinalSetup

```diff
@@ -205,6 +205,14 @@ EOF
     sed -i "s/MAX_RAM=[^;]*/MAX_RAM=${MAX_MEMORY}/" "${FTB_DIR}/settings.cfg"
   fi
 
+  # patch CurseForge settings.sh file, if present
+  if [ -f "${FTB_DIR}/settings.sh" ]; then
+    sed -i "s/MIN_RAM=[^;]*/MIN_RAM=\"${INIT_MEMORY}\"/" "${FTB_DIR}/settings.sh"
+    sed -i "s/MAX_RAM=[^;]*/MAX_RAM=\"${MAX_MEMORY}\"/" "${FTB_DIR}/settings.sh"
+    ESCAPED_JVM_XX_OPTS=$(echo "$JVM_XX_OPTS" | sed -e 's/\//\\\//g')
+    sed -i "s/JAVA_PARAMETERS=[^;]*/JAVA_PARAMETERS=\"${ESCAPED_JVM_XX_OPTS//[$'\t\r\n']} ${JVM_OPTS} $expandedDOpts\"/" "${FTB_DIR}/settings.sh"
```

Comment:
I wonder if you should just leave the JAVA_PARAMETERS as provided by the modpack; however, I do like that this change offers the most flexibility.

Reply:
The modpack's JAVA_PARAMETERS (in this case), is a simplification of the Aikar flags. It sets G1GC and a few options.

> I wonder if you should just leave the JAVA_PARAMETERS as provided by the modpack

That's what I was thinking too, but at the same time, it feels weird to have the options to change these values and them not apply only for this server.
Maybe a "recommendation" could be put in the README when using the CurseForge type that tells the user to enable the Aikar flags?

Reply:
`HU`